.github/ISSUE_TEMPLATE/code_quality.md

```diff
@@ -21,8 +20,3 @@ title: "<Replace this with the actual title>"
 
 <!-- Add any other context suggestion here. -->
 
```

Comment:
Please remove this blank line. That is causing the [CI to fail](https://github.com/WordPress/openverse-catalog/actions/runs/4110774618/jobs/7093917647#step:7:52).

You can also run lint locally to ensure that this file is cleaned up before pushing. For that you will need to run these two commands:

```bash
$ just install
$ just lint
```